experimental/packages/opentelemetry-api-metrics/src/index.ts

```diff
@@ -14,13 +14,45 @@
  * limitations under the License.
  */
 
-export * from './NoopMeter';
-export * from './NoopMeterProvider';
-export * from './types/Meter';
-export * from './types/MeterProvider';
-export * from './types/Metric';
-export * from './types/ObservableResult';
+export {
+  NOOP_METER,
```

Comment:
Do we need to export noop instances? I didn't find noop instances are exported in `@opentelemetry/api`.

Reply:
Yes, looks like it is used in the MeterProvider (returned for new meters when the provider is shutdown) and in the `@opentelemetry/node-sdk` package. I'll see if I can apply some refactoring to make it more similar to `@opentelemetry/api`.

Reply:
Instead of exporting the NOOP instance itself we could export an API to get a noop meter. This would avoid that people rely on existence of such an instance or start to compare to it. it would be also possible to change the implementation to return a fresh Noop instead always the same in case it turns out that this is needed.

Reply:
That's a good idea. I added this in [c8cba2e](https://github.com/open-telemetry/opentelemetry-js/pull/3198/commits/c8cba2e088ec71a3267e1f5547839206e604638c). :slightly_smiling_face: